pkg/controller/backup_status_updater.go

```diff
@@ -436,6 +452,57 @@ func updateBRProgress(progresses []v1alpha1.Progress, step *string, progress *fl
 	return progresses, isUpdate
 }
 
+func updateBackoffRetryStatus(status *v1alpha1.BackupStatus, newStatus *BackupUpdateStatus) bool {
```

Comment:
can you make you branches' condition human readable?

Reply:
ok